doc/whats_new/v0.21.rst

```diff
@@ -56,6 +56,15 @@ Support for Python 3.4 and below has been officially dropped.
   order for further speed performances. :issue:`12251` by `Tom Dupre la Tour`_.
 
 
+:mod:`sklearn.tree`
+...................
+- |Feature| ``get_n_leaves()`` and ``get_max_depth()`` have been added to
+  :class:`tree.BaseDecisionTree` and consequently all estimators based
+  on it, including :class:`tree.DecisionTreeClassifier`,
+  :class:`tree.DecisionTreeRegressor`, :class:`tree.ExtraTreeClassifier`,
+  and :class:`tree.ExtraTreeRegressor`.
+  :issue:`7613` by :user:`Adrin Jalali <adrinjalali>`.
```

Comment:
issue should link to your PR #12300, not the actual issue

Reply:
nah, not really. There seems to be different preferences on this one, and I personally prefer to link to the issues and not the PRs, and if you check v0.20 [whats_new](http://scikit-learn.org/stable/whats_new.html#version-0-20) page, there are 352 cases of links with "/issue/" in them, and only 1 "/pull/".

Reply:
And if you click on most of them, those /issue/ links actually lead to a PR. It's misleading and I did the same before (reference an issue number), but was corrected by @glemaitre

Reply:
Confusing but :issue: link to the PR (The PR will link to the issue)

Reply:
The advantage of linking to the pr is:
* It's easier for us to track the difference between the change log and the commit log
*  it's easier to find your way to all relevant history

sloria's Sphinx-issues library which we vendor here recently supports a separate :pr: role so we could upgrade and start using that.

Reply:
So should it be sth like:

```
- |Feature| :issue:`7613` ``get_n_leaves()`` and ``get_max_depth()`` have been added to
  :class:`tree.BaseDecisionTree` and consequently all estimators based
  on it, including :class:`tree.DecisionTreeClassifier`,
  :class:`tree.DecisionTreeRegressor`, :class:`tree.ExtraTreeClassifier`,
  and :class:`tree.ExtraTreeRegressor`.
  :pr:`12300` by :user:`Adrin Jalali <adrinjalali>`.
```